test/background_test.rb

```diff
@@ -0,0 +1,38 @@
+require 'test_helper'
+
+class BackgroundTest < Minitest::Test
+  include InvoicePrinterHelpers
+
+  def setup
+    @file_name = 'deleteme.pdf'
```

Comment:
Let's avoid tests needed this for now. I have in mind integration tests that will actually compare the output/examples and that the features that way (so it's not just about exceptions). So test only InvoicePrinter.render and feel free to not test InvoicePrinter.print for this.